projects/lib/src/lib/dynamic-form-material.module.ts

```diff
@@ -43,15 +47,34 @@ import {
   imports: [
     MatButtonModule,
     MatCheckboxModule,
+    MatDatepickerModule,
     MatFormFieldModule,
     MatIconModule,
     MatInputModule,
     MatRadioModule,
     MatSelectModule,
     MatOptionModule,
 
+    NativeDateModule,
+
     ReactiveFormsModule,
     CommonModule
+  ],
+  providers: [
+    { provide: MAT_DATE_LOCALE, useValue: 'en-GB' },
```

Comment:
That shouldn't be in the library. Locale settings are per app, not per library. Remove.